File: 007/7.Soru.py
import math

def asalMi(sayi):
    if sayi == 1:
        return False
    if sayi == 2:
        return True
    for i in range(2, int(math.sqrt(sayi)) + 1):
        if sayi % i == 0:
            return False
    return True

sayac, sayi = 0, 1
while sayac < 10001:
    sayi += 1
    if asalMi(sayi):
        sayac += 1

print(str(sayac) + ". asal sayı = " + str(sayi))

File: 007/7.Soru.cs
using System;

public class HelloWorld
{
    static bool asalMi(long sayi)
    {
        long sayac = 0;
        if(sayi==1)
            return false;
        else if(sayi==2)
            return true;
        for(long i=1;i<sayi;i++)
        {
            if(sayi % i == 0)
                sayac++;
        }
        if(sayac == 0)
            return true;
        else
            return false;
    }
    
    public static void Main(string[] args)
    {
        long sayac = 0,sayi=1;
        do{
            if(asalMi(sayi)==true)
                sayac++;
        }while(sayac==10001);

        Console.WriteLine(sayac+". asal sayı = "+sayi);
    }
}
